src/index.js

```diff
@@ -1,5 +1,10 @@
 import React from "react";
 import ReactDOM from "react-dom/client";
+import Router from "./Router";
+import "./style/variables.scss";
+import "./style/mixin.scss";
+import "./style/reset.scss";
+import "./style/variables.scss";
```

Comment:
```suggestion
import "./style/reset.scss";
import "./style/common.scss";
```

variables와 mixin은 해당 스타일이 사용되는 위치에서만
import 하셔서 사용하는 것이 좋습니다!